README.md

```diff
@@ -1,146 +1,272 @@
+- :page_facing_up: GPT-like Large Language Models (LLMs)  
+- :mag_right: CLIP  
+- :scissors: Segment Anything (SAM)  
+- :heavy_plus_sign: And many more
 
-Fondant focuses on data preparation to fine-tune these models.
+## :mouse_trap: Why Fondant?
 
-## Installation
+Foundation models simplify inference by solving multiple tasks across modalities with a simple
+prompt-based interface. But what they've gained in the front, they've lost in the back. 
+**These models require enormous amounts of data, moving complexity towards data preparation**, and 
+leaving few parties able to train their own models.
 
-Fondant can be installed from source using pip:
+We believe that **innovation is a group effort**, requiring collaboration. While the community has 
+been building and sharing models, everyone is still building their data preparation from scratch.
+**Fondant is the platform where we meet to build and share data preparation workflows.**
 
-```
-pip install git+https://github.com/ml6team/fondant.git
-```
+Fondant offers a framework to build **composable data preparation pipelines, with reusable 
+components, optimized to handle massive datasets**. Stop building from scratch, and start 
+leveraging reusable components instead to:
+- Extend your data with public datasets
+- Generate new modalities using captioning, segmentation, translation, image generation, ...
+- Distill knowledge from existing foundation models
+- Filter out low quality data
+- Deduplicate data
 
-## Introduction
+And create high quality datasets to fine-tune your own foundation models.
 
-Fondant is built upon [KubeFlow pipelines](https://www.kubeflow.org/docs/components/pipelines/), a cloud-agnostic and open-source framework built by Google to run machine learning workflows on Kubernetes.
+<p align="right">(<a href="#chocolate_bar-fondant">back to top</a>)</p>
 
-Fondant consists of a pipeline that runs a sequence of components (steps) one after the other. Each step implements one logical piece of a data processing pipeline, like image filtering, image embedding, text deduplication, etc.
+## :magic_wand: Example pipelines
 
-A Fondant pipeline always includes a loading component as first component, followed by one or more transform components. The loading component loads some initial seed data, with the transform components making transformations on them (like enrichment, filtering, transformation).
+Curious to see what Fondant can do? Have a look at our example pipelines:
 
-## Fondant Components
+### Fine-tuning ControlNet
+
+Our 
+[example pipeline to generate data for ControlNet fine-tuning](examples/pipelines/controlnet-interior-design) 
+allows you to create models that you can control using inpainting, segmentation, and 
+regeneration. All you need to get started is a set of prompts describing the type of images to 
+generate.
+
+Using our ControlNet model fine-tuned on interior design images for instance allows you to generate 
+the room of your dreams:
```

Comment:
This sentence was a bit harder to read, 2 alternatives:
- "For instance, using our ControlNet model fine-tuned on interior design images, allows you to generate the room of your dreams:"
- "For instance, fine-tuning ControlNet on interior design images allows you to generate the room of your dreams:"